pandas/util/testing.py

```diff
@@ -934,6 +934,7 @@ def assert_attr_equal(attr, left, right, obj='Attributes'):
         Specify object name being compared, internally used to show appropriate
         assertion message
     """
+    __tracebackhide__ = True
```

Comment:
Can you show the difference in output if adding this?

Reply:
That might be interesting to see to what extent we want to do this, also for the other assert functions. But can you leave that for a separate PR?

Reply:
Fair enough